security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -8,24 +8,48 @@ toc::[]
 
 == Purpose
 
-Machine Config Operator certificates are used to secure connections between the Red Hat Enterprise Linux CoreOS (RHCOS) nodes and the Machine Config Server.
+This certificate authority is used to secure connections from nodes to Machine Config Server (MCS) during initial provisioning.
+
+There are two certificates:
+. A self-signed CA, the MCS CA
+. A derived certificate, the MCS cert
+
+=== Provisioning details
+
+{product-title} installations using {op-system-first} are installed using Ignition. This process is split into two parts:
+
+. An Ignition config is created which is references a URL for the full configuration served by the MCS.
+. The Ignition config manifests as a `worker.ign` file created by the `openshift-install` command for user-provisioned infrastucture installation methods. For installer-provisioned infrastructure installation methods that use the Machine API Operator, this configuration appears as the `worker-user-data` secret.
```

Comment:
```suggestion
. For user-provisioned infrastructure installation methods, the Ignition config manifests as a `worker.ign` file created by the `openshift-install` command. For installer-provisioned infrastructure installation methods that use the Machine API Operator, this configuration is displayed as the `worker-user-data` secret.
```
I'm not completely sure what "appears" meant in the original text, but I replaced it with "is displayed" per IBM Style, which warns against using "appear" instead of another term, such as "opens" or "is displayed". If "appears" is the best word to use here considering the context, feel free to use it.